.github/scripts/fxcore-sync-up-version.js

```diff
@@ -25,15 +25,15 @@ else if(templateVersion.includes("rc")){
     templateConfigFile.version = "0.0.0-rc";
     fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4))
 }
-else if(templateVersion.includes("beta")) {
-    console.log("sync up template in fx-core as 0.0.0-beta")
-    templateConfigFile.version = "0.0.0-beta";
-    templateConfigFile.tagPrefix = "templates-";
-    fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4))
-}
 else if(templateVersion.includes("alpha")) {
     console.log("sync up template in fx-core as 0.0.0-alpha")
     templateConfigFile.version = "0.0.0-alpha";
     templateConfigFile.tagPrefix = "templates-";
     fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4))
+}
+else if(templateVersion.includes("prerelease")) {
```

Comment:
TBD, Here I still version the prerelease with preid as alpha, I should discuss with you about this.